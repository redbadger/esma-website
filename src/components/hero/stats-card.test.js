import React from "react";
import renderer from "react-test-renderer";
import StatsCard from "./stats-card";

describe("StatsCard", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<StatsCard styles={{}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
