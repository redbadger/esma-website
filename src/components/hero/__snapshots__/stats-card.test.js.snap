// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsCard renders correctly 1`] = `
<div
  className="carousel-root"
  tabIndex={0}
>
  <div
    className="carousel carousel-slider"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <div
      className="slider-wrapper axis-horizontal"
      style={Object {}}
    >
      <ul
        className="slider animated"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        style={
          Object {
            "MozTransform": "translate3d(-100%,0,0)",
            "MozTransitionDuration": "350ms",
            "MsTransform": "translate3d(-100%,0,0)",
            "MsTransitionDuration": "350ms",
            "OTransform": "translate3d(-100%,0,0)",
            "OTransitionDuration": "350ms",
            "WebkitTransform": "translate3d(-100%,0,0)",
            "WebkitTransitionDuration": "350ms",
            "msTransform": "translate3d(-100%,0,0)",
            "msTransitionDuration": "350ms",
            "transform": "translate3d(-100%,0,0)",
            "transitionDuration": "350ms",
          }
        }
      >
        <li
          className="slide"
          onClick={[Function]}
        >
          <div
            css={
              Object {
                "map": undefined,
                "name": "fmlnap",
                "next": undefined,
                "styles": "
  height:14rem;--bg-opacity:1;background-color:rgba(255, 255, 255, var(--bg-opacity));border-top-width:8px;--border-opacity:1;border-color:rgba(55, 114, 255, var(--border-opacity));

  div {
    margin-left:0.75rem;text-align:left;@media (min-width: 768px){margin-left:1.5rem;}
  }
",
                "toString": [Function],
              }
            }
          >
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1xwz4nc",
                  "next": undefined,
                  "styles": "
  font-size:3rem;margin-top:0.75rem;margin-bottom:0.75rem;font-weight:300;@media (min-width: 768px){margin-top:1.5rem;}
",
                  "toString": [Function],
                }
              }
            >
              64%
            </div>
            <div>
              People from a working class background earn 64% less a year than someone with a professional background
            </div>
          </div>
        </li>
        <li
          className="slide selected"
          onClick={[Function]}
        >
          <div
            css={
              Object {
                "map": undefined,
                "name": "fmlnap",
                "next": undefined,
                "styles": "
  height:14rem;--bg-opacity:1;background-color:rgba(255, 255, 255, var(--bg-opacity));border-top-width:8px;--border-opacity:1;border-color:rgba(55, 114, 255, var(--border-opacity));

  div {
    margin-left:0.75rem;text-align:left;@media (min-width: 768px){margin-left:1.5rem;}
  }
",
                "toString": [Function],
              }
            }
          >
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1xwz4nc",
                  "next": undefined,
                  "styles": "
  font-size:3rem;margin-top:0.75rem;margin-bottom:0.75rem;font-weight:300;@media (min-width: 768px){margin-top:1.5rem;}
",
                  "toString": [Function],
                }
              }
            >
              24%
            </div>
            <div>
              People from a working class background earn 24% less a year than someone with a professional background
            </div>
          </div>
        </li>
        <li
          className="slide"
          onClick={[Function]}
        >
          <div
            css={
              Object {
                "map": undefined,
                "name": "fmlnap",
                "next": undefined,
                "styles": "
  height:14rem;--bg-opacity:1;background-color:rgba(255, 255, 255, var(--bg-opacity));border-top-width:8px;--border-opacity:1;border-color:rgba(55, 114, 255, var(--border-opacity));

  div {
    margin-left:0.75rem;text-align:left;@media (min-width: 768px){margin-left:1.5rem;}
  }
",
                "toString": [Function],
              }
            }
          >
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1xwz4nc",
                  "next": undefined,
                  "styles": "
  font-size:3rem;margin-top:0.75rem;margin-bottom:0.75rem;font-weight:300;@media (min-width: 768px){margin-top:1.5rem;}
",
                  "toString": [Function],
                }
              }
            >
              42%
            </div>
            <div>
              People from a working class background earn 42% less a year than someone with a professional background
            </div>
          </div>
        </li>
        <li
          className="slide"
          onClick={[Function]}
        >
          <div
            css={
              Object {
                "map": undefined,
                "name": "fmlnap",
                "next": undefined,
                "styles": "
  height:14rem;--bg-opacity:1;background-color:rgba(255, 255, 255, var(--bg-opacity));border-top-width:8px;--border-opacity:1;border-color:rgba(55, 114, 255, var(--border-opacity));

  div {
    margin-left:0.75rem;text-align:left;@media (min-width: 768px){margin-left:1.5rem;}
  }
",
                "toString": [Function],
              }
            }
          >
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1xwz4nc",
                  "next": undefined,
                  "styles": "
  font-size:3rem;margin-top:0.75rem;margin-bottom:0.75rem;font-weight:300;@media (min-width: 768px){margin-top:1.5rem;}
",
                  "toString": [Function],
                }
              }
            >
              64%
            </div>
            <div>
              People from a working class background earn 64% less a year than someone with a professional background
            </div>
          </div>
        </li>
        <li
          className="slide selected"
          onClick={[Function]}
        >
          <div
            css={
              Object {
                "map": undefined,
                "name": "fmlnap",
                "next": undefined,
                "styles": "
  height:14rem;--bg-opacity:1;background-color:rgba(255, 255, 255, var(--bg-opacity));border-top-width:8px;--border-opacity:1;border-color:rgba(55, 114, 255, var(--border-opacity));

  div {
    margin-left:0.75rem;text-align:left;@media (min-width: 768px){margin-left:1.5rem;}
  }
",
                "toString": [Function],
              }
            }
          >
            <div
              css={
                Object {
                  "map": undefined,
                  "name": "1xwz4nc",
                  "next": undefined,
                  "styles": "
  font-size:3rem;margin-top:0.75rem;margin-bottom:0.75rem;font-weight:300;@media (min-width: 768px){margin-top:1.5rem;}
",
                  "toString": [Function],
                }
              }
            >
              24%
            </div>
            <div>
              People from a working class background earn 24% less a year than someone with a professional background
            </div>
          </div>
        </li>
      </ul>
    </div>
    <ul
      className="control-dots"
    >
      <li
        aria-label="slide item 1"
        className="dot selected"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
        value={0}
      />
      <li
        aria-label="slide item 2"
        className="dot"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
        value={1}
      />
      <li
        aria-label="slide item 3"
        className="dot"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
        value={2}
      />
    </ul>
    <p
      className="carousel-status"
    />
  </div>
</div>
`;
